crates/adapters/src/integrated/delta_table/input.rs

```diff
@@ -273,23 +289,72 @@ impl DeltaTableInputEndpointInner {
         }
     }
 
+    /// Compute the subset of columns in the Delta table schema that occur in the SQL
+    /// table declaration.
+    fn relevant_columns(&self, table: &DeltaTable) -> Vec<String> {
```

Comment:
I don't know if "relevant" is the right word.
"accessed"?

Reply:
I'll call them `used_columns`.